SwiftUI v2

chriseidhof opened the pull request.

chriseidhof committed:
Chapter 2 Exercises

chriseidhof committed:
Knob edits

chriseidhof committed:
Simplify

chriseidhof committed:
Simplify

chriseidhof committed:
Take 2

chriseidhof committed:
Double frame

JohnCoates commented:
Why the double frame in `Chapter 05/TabView2/TabView/ContentView.swift`?
Why not just `.frame(height: 2, alignment: .bottom)`?

chriseidhof commented:
@JohnCoates sorry for the late reply. That works as well and is simpler, thank you! I think we added the double frame before we added the matched geometry effect, that's probably the reason!

chriseidhof committed:
Simplify

malhal commented:
Should be @StateObject here https://github.com/objcio/thinking-in-swiftui-sample-code/blob/53e6cd1a31a41c446d180af742792439d3840323/Chapter%2002/SimpleClock/SimpleClock/ContentView.swift#L38

But given Text has a built-in timer object that even updates for locale change I'm not sure is a good sample.

floriankugler committed:
Grids

floriankugler committed:
Grid example updated

floriankugler committed:
Matched Geometry Effect

floriankugler committed:
Matched geometry edits

floriankugler committed:
Remove iflet helper
